### DISCUSSION TITLE
How to Authenticate with AKS

### DISCUSSION THREAD

**Discussion by jptissot** [kubernetes-client/csharp#1129] [Q&A]
Is it possible to use the `Azure.Identity` package to get the `KubernetesClientCredentials` token.

**Comment by IvanJosipovic** [comment]
See this post for some example code, though there appear to be issues. https://github.com/kubernetes-client/csharp/issues/1113

**Reply by jptissot** [reply]
Thanks, Added a comment there for what worked for me. Any comments there?

**Reply by tg123** [reply]
i will see if i can pull aks/aad guys to take a look
